internal/envparser/envparser.go

```diff
@@ -113,7 +113,7 @@ func (p *lineParser) parse(envVars *EnvVars, l string) error {
 
 	// key is first index, value is second
 	k := strings.TrimSpace(splits[0])
-	v := strings.TrimSpace(splits[1])
```

Comment:
Is this going to affect multi line strings?

Reply:
Wouldn't this test fail if it did? https://github.com/wingocard/serum/blob/master/internal/envparser/envparser_test.go#L128